bing_bert/deepspeed_train.py

```diff
@@ -390,7 +390,7 @@ def prepare_optimizer_parameters(args, model):
 
 def prepare_model_optimizer(args):
     # Initialize torch distributed
-    # torch.distributed.init_process_group(backend="nccl")
+    deepspeed.init_distributed(dist_backend='nccl')

```

Comment:
How was this working previously? It seems process group initialization was not happening?

Reply:
In this case deepspeed.initialize(..) was and still will do it if it's not done explicitly.  However, I uncommented it here to show how to do it as an example. We used to require torch distributed initialized to instantiate the bert model (we've since removed that requirement though).

Reply:
Okay, I think I see it now. DS Engine will initialize by default if client does not explicitly forbid.